tests/charm++/varTRAM/vartest.ci

```diff
@@ -0,0 +1,15 @@
+mainmodule vartest {
+  readonly CProxy_main mainProxy;
+  mainchare main {
+    entry main(CkArgMsg*);
+    entry [reductiontarget] void startsum(int);
+    entry [reductiontarget] void done(int);
+  };
+  array [1D] engine {
+    entry engine();
+    entry engine(int,int,int);
+    entry void simulate();
+    entry [aggregate(bufferSize : 2048, numDimensions : 2, thresholdNum : 1, thresholdDen : 2, cutoffNum : 1, cutoffDen : 2)] void ping(msg);
```

Comment:
I think 'Numer' and 'Denom' would be more intuitive than 'Num' and 'Den'. I assumed Num meant Number and I didn't know what Den was until I read through the charmxi code changes. Maybe something to discuss in Core meeting.

Reply:
Could 'msg' just be std::vector or does it have to be a message type?

Reply:
Can you still combine aggregate(...) with other entry method attributes? Such as nokeep or inline or something.

Reply:
msg can be a std::vector, since it has a PUPer. I've tested, it is compatible with other entry method attributes.

Reply:
Can you make it a std:vector then? It would be cleaner